x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/index.mock.ts

```diff
@@ -28,7 +28,7 @@ const helpersMock: jest.Mocked<CasesService['helpers']> = {
   groupAlertsByRule: jest.fn(),
 };
 
-export const createCasesServiceMock = (): CasesService => ({
```

Comment:
I must have removed this because I wanted to use the original services type from Cases but ended up keeping the TAUI specific type to avoid dependencies between the plugins. I'll add it back!